docs/.island/config.ts

```diff
@@ -125,7 +125,11 @@ function getSidebar(lang: 'zh' | 'en') {
           {
             text: getText('静态资源', 'Static Assets'),
             link: getLink('/guide/static-assets')
-          }
+          },
+          {
```

Comment:
I think these content should be put into `Using mdx` article instead because it belongs to mardown grammer.